### ISSUE TITLE
Bypass GitHub cache

### ISSUE THREAD

**Issue by tj** [tj/sponsors-api#1]
I recall GitHub caching Readme images aggressively, but I forget what was required there, need to research again, maybe just a `Cache-Control: no-cache`

**Comment by dzikoysk** [comment]
Yup, we use these to properly display badges on GitHub in README etc.:

* [MavenLatestApiEndpoints#L156](https://github.com/dzikoysk/reposilite/blob/aadad3e02ef004d06e3fd782b9d87c59732b0405/reposilite-backend/src/main/kotlin/com/reposilite/maven/infrastructure/MavenLatestApiEndpoints.kt#L156-L158)
